WordPress/gutenberg
# [Mobile] Fix gallery style issues

Related gutenberg-mobile PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/1766

## Description
This PR fixes some known style issues listed here: https://github.com/wordpress-mobile/gutenberg-mobile/issues/1674#issue-536788672. It adjusts the appender color in the gallery block, and fixes a fence-post issue within gallery settings BottomSheet.

## How has this been tested?
* Select a gallery block with images
* Make sure the appender (+) color matches the settings (gear) icon color
* Tap the gear icon
* Last cell in gallery settings bottom sheet shouldn’t have a separator (easier to see on iOS)

## Screenshots 

Example of appender color mismatch:

![image](https://user-images.githubusercontent.com/8507675/71260165-e9e5de80-2385-11ea-8992-85edcfd8312a.png)

Example of separator problem:

![image](https://user-images.githubusercontent.com/8507675/70687806-e99d7180-1cfb-11ea-9e9e-1d1c2996c308.png)
</details>


## Types of changes
A mobile specific prop constant is introduced to specify separator type = 'none' for the last item in the `InspectorControls`. Since the last item can vary depending on whether Image size options are displayed, the control preceding "Images Size" control (i.e. "Link To") will conditionally have its separator type set to 'none'. Ideally, this fence-post logic would not be necessary here, and could be handled somewhere else, such as the controls' container, however, this may require more investigation.

## Checklist:
